assets/javascripts/discourse/templates/components/babble-icon.hbs

```diff
@@ -1,6 +1,6 @@
-{{#unless noTopicAvailable}}
+{{#if babbleEnabled}}
   {{#header-dropdown iconId="babble-notifications"
-                     icon="bullhorn"
+                     icon=Discourse.SiteSettings.babble_icon
```

Comment:
Hey @JSey, there's a deprecation around using Discourse.SiteSettings in the view, can we do something like
```
icon=babbleIcon
```
and in the controller
```
self.set('babbleIcon', Discourse.SiteSettings.babble_icon)
```
?